### DISCUSSION TITLE
Need to update EFI for catalina install

### DISCUSSION THREAD

**Discussion by stjuki** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#337] [Q&A]
I am trying to install Catalina and the network utility shows the adapter but it does not seem to show it as a wifi adapter.  Also, there is nowhere on the bar where I can select Wifi Networks.  Please help. I have attached my EFI

System is
hackintosh optiplex 3020
i5-4570
igpu HD4600

Wifi card: T3U Plus

Thank you in advance.
[EFI.zip](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/files/14908424/EFI.zip)

**Comment by chris1111** [comment]
This topic is not  relate to install macOS

**Reply by stjuki** [reply]
Sorry and thank you...
